From 0eb15ac98d0d80a3e7a44a70cedea74d30d3da2f Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 13:11:23 -0400
Subject: [ethernet] i40e/i40evf: Add an FD message level

Message-id: <1409663591-32531-41-git-send-email-sassmann@redhat.com>
Patchwork-id: 90519
O-Subject: [RHEL7.1 PATCH 040/148] i40e/i40evf: Add an FD message level
Bugzilla: 1091127
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Anjali Singhai Jain <anjali.singhai@intel.com>

To use for Flow Director specific messages.

Change-ID: I69e39a410aa2661f8fd1ed6af0126fa4c335cb77
Signed-off-by: Anjali Singhai Jain <anjali.singhai@intel.com>
Acked-by: Shannon Nelson <shannon.nelson@intel.com>
Acked-by: Kevin Scott <kevin.c.scott@intel.com>
Signed-off-by: Catherine Sullivan <catherine.sullivan@intel.com>
Tested-by: Kavindya Deegala <kavindya.s.deegala@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit c2e1b5966a7deb08e44276fa1429175565a7f02d)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40evf/i40e_type.h b/drivers/net/ethernet/intel/i40evf/i40e_type.h
index efe73ad..4673b33 100644
--- a/drivers/net/ethernet/intel/i40evf/i40e_type.h
+++ b/drivers/net/ethernet/intel/i40evf/i40e_type.h
@@ -88,6 +88,7 @@ enum i40e_debug_mask {
  I40E_DEBUG_FLOW   = 0x00000200,
  I40E_DEBUG_DCB   = 0x00000400,
  I40E_DEBUG_DIAG   = 0x00000800,
+ I40E_DEBUG_FD   = 0x00001000,
 
  I40E_DEBUG_AQ_MESSAGE  = 0x01000000,
  I40E_DEBUG_AQ_DESCRIPTOR = 0x02000000,
-- 
1.7.1